lib/base_helper.rb

```diff
@@ -392,6 +392,15 @@ def find_follow(*paths)
         end
       end
 
+      def get_oauthaccesstoken(token)
+        if token.include? "sha256~"
+          oauthaccesstoken = token.gsub("sha256~","")
+          oauthaccesstoken = "sha256~" + Base64.urlsafe_encode64(OpenSSL::Digest::SHA256.digest(oauthaccesstoken),padding: false)
+        else
+          raise "provided user token is not valid"
```

Comment:
From good code perspective, add several words to explicitly tell for which it is not valid.